README.md

```diff
@@ -0,0 +1 @@
+test
```

Comment:
This seems to be quite... empty.... for such a lovely project..... 😄

Reply:
Yeah you are right!